ui/job-view/headerbars/PrimaryNavBar.jsx

```diff
@@ -73,14 +83,14 @@ export default class PrimaryNavBar extends React.Component {
 }
 
 PrimaryNavBar.propTypes = {
+  updateButtonClick: PropTypes.func.isRequired,
```

Comment:
This is just reordering the ``propTypes`` so that all the ``func``s are at the beginning.  They don't need to get checked in ``shouldComponentUpdate`` so thiis makes it visually a little easier to verify all the right fields are getting checked.